test/integration/expel_test.lua

```diff
@@ -38,8 +38,6 @@ g.before_all(function()
             uuid = ...,
             expelled = true,
         }}})
-
-        box.space._cluster.index.uuid:delete(...)
```

Comment:
NIT. Whole eval can be rewritten with a shorter `netbox.call`. Something like this:

```lua
    g.A1.net_box:eval(
        'package.loaded.cartridge.admin_edit_topology',
        {{servers = {{uuid = expelled.instance_uuid, expelled = true}}}}
    )
```